src/pkg/k8s/images.go

```diff
@@ -19,7 +20,7 @@ type ImageMap map[string]bool
 type ImageNodeMap map[string][]string
 
 // GetAllImages returns a list of images and their nodes found in pods in the cluster.
-func (k *K8s) GetAllImages(timeoutDuration time.Duration) (ImageNodeMap, error) {
+func (k *K8s) GetAllImages(timeoutDuration time.Duration, requestedCPU resource.Quantity, requestedMemory resource.Quantity) (ImageNodeMap, error) {
```

Comment:
```suggestion
func (k *K8s) GetAllImages(timeoutDuration time.Duration, minNodeCPU resource.Quantity, minNodeMemory resource.Quantity) (ImageNodeMap, error) {
```
Another name suggestion but minNodeCPU may help folks better understand what these are if they're using them as a library.